src/alma-gateway-for-woocommerce.php

```diff
@@ -50,18 +50,111 @@
 	define( 'ALMA_PLUGIN_URL', plugin_dir_url( __FILE__ ) );
 }
 
+// Define a constant to see if site is network activated.
+if ( ! function_exists( 'is_plugin_active_for_network' ) ) {
+	// Makes sure the plugin is defined before trying to use it.
+	require_once ABSPATH . '/wp-admin/includes/plugin.php';
+}
+
+if ( is_plugin_active_for_network( 'alma-gateway-for-woocommerce/alma-gateway-for-woocommerce.php' ) ) {
+	define( 'ALMA_PLUGIN_NETWORK_ACTIVATED', true );
+} else {
+	define( 'ALMA_PLUGIN_NETWORK_ACTIVATED', false );
+}
+
+/**
+ * Retrieve an option value from the current network or site.
+ *
+ * @param string $option Name of the option to retrieve.
+ * @param mixed  $default The default value.
+ * @param bool   $deprecated Wheter to use cache.
+ *
+ * @return false|mixed|null
+ */
+function alma_get_option( $option, $default = false, $deprecated = true ) {
+
+	if ( ALMA_PLUGIN_NETWORK_ACTIVATED ) {
+		// Get network site option.
+		return get_site_option( $option, $default, $deprecated );
+	}
+	// Get site option.
+	return get_option( $option, $default );
+}
+
+/**
+ * Update an existing option.
+ *
+ * @param string  $option Name of the option to retrieve.
+ * @param mixed   $value The value.
+ * @param boolean $autoload Autoload.
+ *
+ * @return bool
+ */
+function alma_update_option( $option, $value, $autoload = null ) {
+
+	if ( ALMA_PLUGIN_NETWORK_ACTIVATED ) {
+		// Get network site option.
+		return update_site_option( $option, $value );
+	}
+	// Get site option.
+	return update_option( $option, $value, $autoload );
+}
+
+/**
+ * Delete an option.
+ *
+ * @param string $option Name of the option to delete.
+ *
+ * @return bool
+ */
+function alma_delete_option( $option ) {
+
+	if ( ALMA_PLUGIN_NETWORK_ACTIVATED ) {
+		// Get network site option.
+		return delete_site_option( $option );
+	}
+	// Get site option.
+	return delete_option( $option );
```

Comment:
I think the method name is `alma_delete_option`  ?

Reply:
Not in this case